src/PCA9633.h

```diff
@@ -195,7 +195,9 @@
  */
 #define BIT_LDR0    0
 
-
+// LED driver output type, OUTDRV (page 11, MODE2 register table)
+#define OUTDRV_OPEN_DRAIN 0
```

Comment:
Please add:
/**
 \* The 4 LED outputs are configured with an open-drain structure
 */